sys/freebsd/aio.txt

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <aio.h>
+include <sys/uio.h>
+
+aio_cancel(fildes fd, iocb ptr[in, aiocb])
+aio_error(iocb ptr[in, aiocb])
+aio_read(iocb ptr[in, aiocb])
+aio_readv(iocb ptr[in, aiocb])
+aio_return(iocb ptr[in, aiocb])
+aio_suspend(iocbs ptr[in, array[aiocb]], niocb len[iocbs], timeout ptr[in, timespec])
+aio_waitcomplete(iocbp ptr[in, aiocb], timeout ptr[in, timespec])
+aio_write(iocb ptr[in, aiocb])
+aio_writev(iocb ptr[in, aiocb])
+aio_fsync(op int32, iocb ptr[in, aiocb])
+aio_mlock(iocb ptr[in, aiocb])
+lio_listio(mode flags[lio_mode], list ptr[in, array[aiocb]], nent len[list], sig ptr[in, sigevent])
+
+aiocb {
+	aio_fildes	fd
+	aio_offset	int64
+	aio_buf		ptr[in, array[int8]]
+	aio_nbytes	int32
+	spare		array[int32, 2]
+	spare2		ptr[in, array[int8]]
+	aio_lio_opcode	flags[lio_opcodes, int32]
+	aio_reqprio	int32
+	aiocb_private	aiocb_private
+	aio_sigevent	sigevent
```

Comment:
Doesn't need to be addressed in this PR, but the definition of sigevent, in ktimer.txt, is wrong. Specifically the `val` field is not correctly defined.